brainstormforce/astra
# AST-1427 - Toggle style improved for dark mode

### Description
- As per the new layout design for toggle style updated
- https://www.figma.com/file/MELvG4oIUOSH7oBdGe92G1/Astra-Dark-Mode-Toggles?node-id=0%3A1

### Types of changes
- New feature (non-breaking change) 
- HTML markup revampped
- HTML classes and apparently pro CSS reduced

### How has this been tested?
- Check with different switch styles
- Check with design options alterations

### Checklist:
